providers/src/airflow/providers/edge/provider.yaml

```diff
@@ -102,3 +102,11 @@ config:
         type: integer
         example: ~
         default: "60"
+      push_log_chunk_size:
+        description: |
+          Edge Worker uploads log files in chunks. If the log file part which is uploaded exceeds the chunk size it creates a new request. This solves issue if firewall defines max size of request.
+          This value must be defined in Bytes.
```

Comment:
The rationale is usually not a firewall but limits on the application gateway.

Can you point as example to nginx documentation in https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size as nginx is used in most K8s ingress deployments, but also give a hint that if HTTP 413 is seen that this needs to be adjusted.

Reply:
Updated the description.